src/khoj/routers/web_client.py

```diff
@@ -34,19 +34,52 @@
 @web_client.get("/", response_class=FileResponse)
 @requires(["authenticated"], redirect="login_page")
 def index(request: Request):
-    return templates.TemplateResponse("index.html", context={"request": request, "demo": state.demo})
+    user = request.user.object
+    user_picture = request.session.get("user", {}).get("picture")
+
+    return templates.TemplateResponse(
+        "index.html",
+        context={
+            "request": request,
+            "demo": state.demo,
+            "username": user.username if user else None,
```

Comment:
`user` should never be `None` here. Same with the `index_post` method below.
```suggestion
            "username": user.username,
```

Reply:
Valid point. Fixed